packages/rocket-static-sites-aws-infrastructure/src/static-website-stack.ts

```diff
@@ -0,0 +1,76 @@
+import { CfnOutput, RemovalPolicy, Stack } from '@aws-cdk/core'
```

Comment:
Reviewed until here. Long but AMAZING PR. I'll keep reviewing tomorrow.

Reply:
![](https://imgc.allpostersimages.com/img/posters/forever-alone-rage-comic-meme_u-L-PXJB1N0.jpg)

Reply:
LOL, when I said tomorrow, I meant, next week 😂 
Resuming the review!